src/components/ReplyCard/InReplyTo/InReplyToBase.vue

```diff
@@ -62,6 +98,24 @@
       isSelectType() {
         return this.inReplyTo.type === 'application/vnd.lime.select+json'
       },
+      isImageReply() {
+        return this.inReplyTo.value.type === 'image/png'
+      },
+      isVideoReply() {
+        return this.inReplyTo.value.type === 'video/mp4'
+      },
```

Comment:
I think we can have something like "image/jpeg" and the same can happen with video. What you think about checking using includes like the audio checking?

```suggestion
      isImageReply() {
        return this.inReplyTo.value.type.includes('image')
      },
      isVideoReply() {
        return this.inReplyTo.value.type.includes('video')
      },
```

Reply:
@mramosbhz 

I've changed this validation, because some messages dont have the attribute type

`return this.inReplyTo.value.type && this.inReplyTo.value.type.includes('audio')`